dashboard/src/main/resources/webapp/static/lib/angular-topology-editor/canvas.js

```diff
@@ -101,7 +101,8 @@ Graph.Node.popovercontent = function(i) {
 };
 
 var Canvas = (function() {
-
+    var LOG = Log.getLogger("Canvas").setLevel(Log.INFO);
```

Comment:
Because the way AngularJS works I think that probably you can't retrieve in your scripts the variable from loglevel.js as they do in the example:

```
<script src="loglevel.min.js"></script>
<script>
var logging = log.noConflict();

logging.error("still pretty easy");
</script>
```

 I really discourage this kind of implementations that loads global variables in conjunction with AngularJS, it could lead into serious conflicts.